mmv1/templates/terraform/encoders/bigquery_data_transfer.go.tmpl

```diff
@@ -39,6 +39,14 @@ for _, sp := range sensitiveParams {
 		params[sp] = auth.(string)
 	}
 }
+{{- if ne $.Compiler "terraformgoogleconversion-codegen" }}
+for _, sp := range sensitiveWoParams {
+	if auth, _ := d.GetRawConfigAt(cty.GetAttrPath("sensitive_params").IndexInt(0).GetAttr(sp)); !auth.IsNull() {
+		sp = sp[:len(sp)-3] // _wo is convention for write-only params and are removed here
```

Comment:
Thanks, can we add a comment in Config.yaml too? The intention is when people add a new wo field there, they would know something happens in constants/ and encoders/ that requires the new field to be named a certain way and be added to related blocks in those files

Reply:
added